components/console/src/core/infrastructure/midaz/dto/midaz-operation-routes.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { MetadataEntity } from "@/core/domain/entities/metadata-entity"
```

Comment:
While naming the entities with dots is cool, is not the pattern we been following.
Rename this file to "midaz-operation-routes-dto.ts"